### DISCUSSION TITLE
Unable to Register App URL - Enterprise 10.9

### DISCUSSION THREAD

**Discussion by eaustin1** [EsriPS/arcgis-assistant-feedback#35] [Q&A] [answered]
Hello - I am unable to register https://assistant.esri-ps.com to the App Launcher. I am on 10.9 Portal.  Is anyone having this issue? Help!

![image](https://user-images.githubusercontent.com/94077420/141182500-3ec37297-78bd-4e25-9342-faae7fed8b04.png)

**Comment by oevans** [comment, marked as answer]
@eaustin1 -- Have you added ArcGIS Assistant as an item in your portal and registered it? That is a prerequisite to adding an app that is not hosted on the portal. See the doc [here](https://enterprise.arcgis.com/en/portal/latest/administer/windows/manage-app-launcher.htm#ESRI_SECTION1_7D9954DF51FD47DAB4011E786ADF681B).

![Screen Shot 2021-11-10 at 3 04 20 PM](https://user-images.githubusercontent.com/3215530/141185783-1f57f464-ee33-4b9d-9bda-ba499aa1527f.jpg)

Once you do that, Assistant will appear in the list of available apps to add.

![Screen Shot 2021-11-10 at 3 03 28 PM](https://user-images.githubusercontent.com/3215530/141185827-fb445453-0512-4e05-8d32-de5c1d0aa2aa.jpg)

**Reply by oevans** [reply]
The steps to register an app once it's been added as an item are [here](https://enterprise.arcgis.com/en/portal/latest/administer/windows/add-items.htm#ESRI_SECTION1_0D1B620254F745AE84F394289F8AF44B).

**Comment by eaustin1** [comment]
@oevans Thanks! that worked and I was able to register to my org. 
Now when I'm in the assistant - i am unable to login to my portal:

![image](https://user-images.githubusercontent.com/94077420/141190860-cf645a46-3ba4-492f-9d03-ccde3048d26b.png)

![image](https://user-images.githubusercontent.com/94077420/141190884-19572296-9ff8-4a10-a41c-a7a782a47581.png)

**Reply by eaustin1** [reply]
Nevermind - i had to use /arcgis on the end. Thank you! I am in!
